src/main/java/webserver/RequestHandler.java

```diff
@@ -0,0 +1,259 @@
+		UserProcessor.createUser(httpRequestBodyMap);
+
+		byte[] body = Files.readAllBytes(new File("./webapp/index.html").toPath());
+		response302HeaderLocationIndexHtml(dos, body.length);
+		responseBody(dos, body);
+	}
+
+	public Map<String, String> getHttpRequestBodyMap(BufferedReader br, List<String> httpRequestContents) throws IOException {
+		int contentLength = getContentLength(httpRequestContents);
+		String httpRequestBodyString = IOUtils.readData(br, contentLength);
+
+		return HttpRequestUtils.parseRequestBody(httpRequestBodyString);
+	}
+
+	private int getContentLength(List<String> httpRequestContents) {
+		for (String httpRequestContent : httpRequestContents) {
+			if (httpRequestContent.startsWith("Content-Length")) {
+				int contentLengthValueStartIndex = httpRequestContent.indexOf(":") + 1;
+				String contentLengthValueString = httpRequestContent.substring(contentLengthValueStartIndex).trim();
+
+				return Integer.parseInt(contentLengthValueString);
+			}
+		}
+
+		return 0;
+	}
+
+	private String getCookie(List<String> httpRequestContents) {
+		for (String httpRequestContent : httpRequestContents) {
+			if (httpRequestContent.startsWith("Cookie")) {
+				int cookieValueStartIndex = httpRequestContent.indexOf(":") + 1;
+
+				return httpRequestContent.substring(cookieValueStartIndex).trim();
+			}
+		}
+
+		return "";
+	}
+
+	private String getHttpRequestPath(List<String> httpRequestContents) {
+		String httpRequestStartLine = httpRequestContents.get(START_LINE);
+		String httpRequestUrl = getHttpRequestUrl(httpRequestStartLine);
+
+		int queryStringStartIndex = httpRequestUrl.indexOf("?");
+
+		if (queryStringStartIndex == -1) {
+			return httpRequestUrl;
+		}
+
+		return httpRequestUrl.substring(0, queryStringStartIndex);
+	}
+
+	private String getHttpRequestUrl(String httpRequestStartLine) {
+		String[] tokens = httpRequestStartLine.split(" ");
+		return tokens[1];
+	}
+
+	private List<String> getHttpRequestContents(BufferedReader br) throws IOException {
+		List<String> httpRequestContents = new ArrayList<>();
+
+		String line = br.readLine();
+
+		log.info("HttpRequest print start.");
+
+		while (!"".equals(line)) {
+			if (line == null) {
+				return httpRequestContents;
+			}
+
+			httpRequestContents.add(line);
+
+			log.info(line);
+
+			line = br.readLine();
+		}
+
+		log.info("HttpRequest print end.");
+
+		return httpRequestContents;
+	}
+
+	private void response302HeaderLocationIndexHtml(DataOutputStream dos, int lengthOfBodyContent) {
+		try {
+			dos.writeBytes("HTTP/1.1 302 FOUND \r\n");
+			dos.writeBytes("Content-Type: text/html;charset=utf-8\r\n");
+			dos.writeBytes("Content-Length: " + lengthOfBodyContent + "\r\n");
+			dos.writeBytes("Location: /index.html\r\n");
+			dos.writeBytes("\r\n");
+		} catch (IOException e) {
+			log.error(e.getMessage());
+		}
+	}
+
+	private void response302HeaderWithLoginedCookie(DataOutputStream dos, int lengthOfBodyContent, boolean isLoginSuccess, String location) {
```

Comment:
이 메서드도 일반화 할 수 있어 보입니다.
```suggestion
	private void response302Header(DataOutputStream dos, int lengthOfBodyContent, Map<String, List<String>> headers, String location) {
```

Reply:
넵 수정해보겠습니다